pagopa/pn-frontend
# Feature/PN-10910 - Integrate and test Sender Dashboard api

## Short description
This integrates the Sender Dashboard api call into FE to retreive real data from the BE removing the previous mock.

## List of changes proposed in this pull request
- generate the openapi client for the sender dashboard
- fix some issues and test for the proper behaviour 

## How to test
Prerequisite: IS_STATISTICS_ENABLED env variable should be set to true into config.json for PA
- Navigate to the Statistics page and verify the request is sent properly and every chart works as expected
- Login using "Comune di Sappada" as PA to verify the empty state is managed properly

Please note that, since the filtering component has not been implemented yet, the default filter (last 30 days) will be applied to every request automatically.
